startupscript/dataproc/startup.sh

```diff
@@ -734,6 +734,13 @@ chown ${LOGIN_USER}:${LOGIN_USER} "${USER_BASH_PROFILE}"
 ###################################
 # Start workbench app proxy agent 
 ###################################
+# Add a marker for the Terra-specific customizations
+cat << EOF >> "${JUPYTER_CONFIG}"
+
+### BEGIN: Terra-specific customizations ###
```

Comment:
Since this now in a Verily-specific repo, can we make this "VWB-specific customizations"?